src/main/java/edu/iu/terracotta/repository/AssignmentRepository.java

```diff
@@ -6,11 +6,13 @@
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
-import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 
+import javax.transaction.Transactional;
```

Comment:
I'm curious why the switch to the javax Transactional annotation.  I'm not really aware of the pros or cons but I found this (https://stackoverflow.com/questions/26387399/javax-transaction-transactional-vs-org-springframework-transaction-annotation-tr) and one of the answers indicates that you shouldn't mix and match javax Transactional with spring's Transactional annotation, so I just want to make sure this won't cause any unwanted issues down the road.

Reply:
oops! That was an IDE syntax completion selection mistake. I'll fix!